src/trafficcontroller/dopplerproxy/doppler_proxy.go

```diff
@@ -137,13 +137,28 @@ func (proxy *Proxy) serveAppLogs(paths []string, writer http.ResponseWriter, req
 		return
 	}
 
-	authorized, err := proxy.logAuthorize(authToken, appId, proxy.logger)
+	status, err := proxy.logAuthorize(authToken, appId, proxy.logger)
+	if status != http.StatusOK {
+		proxy.logger.Warndf(map[string]interface{}{
+			"status": status,
+			"err":    err,
+		}, "auth token not authorized to access appId [%s].", appId)
+		switch status {
+		case http.StatusUnauthorized:
+			writer.WriteHeader(status)
+			writer.Header().Set("WWW-Authenticate", "Basic")
+			fmt.Fprintf(writer, "You are not authorized.")
+		case http.StatusForbidden:
+			status = http.StatusNotFound
+			fallthrough
+		case http.StatusNotFound:
+			writer.WriteHeader(status)
+			fmt.Fprintf(writer, "App ID is not found.")
+		default:
+			writer.WriteHeader(status)
```

Comment:
Also noticed this morning, if the status code is anything other than the cases listed the response will contain `Internal server error occurred.` For example 201. We could either change the message, check if the status code is >=500, or just write out 500 vs `status`.